src/actions/content.js

```diff
@@ -107,6 +107,35 @@ function* loadContent(action) {
   }
 }
 
+export const CONTENT_SINGLE_REQUESTED = 'CONTENT_SINGLE_REQUESTED';
+export const requestSingleContent = (bundle, id) => ({
+  type: CONTENT_SINGLE_REQUESTED,
+  payload: { bundle, id },
+});
+
+export const CONTENT_SINGLE_LOADED = 'CONTENT_SINGLE_LOADED';
+function* loadSingleContent(action) {
+  const { payload: { bundle, id } } = action;
+  try {
+    yield put(resetLoading());
+    yield put(showLoading());
+
+    const content = yield call(api, 'content_single', { parameters: { bundle, id }, queryString: {
```

Comment:
Should we first look up in state.content.contentById whether the content already exists?